web/libs/editor/src/tags/object/Pdf.jsx

```diff
@@ -37,7 +37,7 @@ const HtxPdf = inject("store")(
   observer(({ item }) => {
     if (!item._url) return null;
     return (
-      <iframe
+      <embed
         src={item._url}
         title="PDF Document"
```

Comment:
that feels like something we don't need. if it's showing up as a popup, that's annoying